saliency/core/guided_ig_test.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the core implementation of Guided IG."""
+import unittest
+
+from . import guided_ig
+from .base import INPUT_OUTPUT_GRADIENTS
+import numpy as np
+
+
+class GuidedIGTest(unittest.TestCase):
```

Comment:
Can you add a test which checks for correctness when baseline > input?

Reply:
Good suggestion. Done.